tests/integration/helpers/cluster.py

```diff
@@ -1757,6 +1757,7 @@ def start_zookeeper_nodes(self, zk_nodes):
             - {db_dir}:/var/lib/clickhouse/
             - {logs_dir}:/var/log/clickhouse-server/
             - /etc/passwd:/etc/passwd:ro
+            - /run/xtables.lock:/run/xtables.lock:ro
```

Comment:
Maybe it worth adding a comment (here and in other places where this file is passed through), AFAIU here it is required only to pass it for the PartitionManager, right?

And also something basic like `pass through lock for iptables for proper --wait` (Like I had in my patch https://github.com/ClickHouse/ClickHouse/pull/28496/commits/5ea852cd53430a5dd5f29fa58076d2fa1346afa0)